create_infotexts.py

```diff
@@ -30,14 +30,15 @@ def load_places_mapping():
     places_df = places_df.set_index("Nyckelord")
     places_df.columns = ["freq", "commonscat", "wikidata"]
 
-    places_df.replace("-", np.nan)
+    places_df[places_df == "-"] = None
```

Comment:
I'm not sure this will actually work though.

What you want to do is go through every entry and if the value is `"-"` then replace it by `None`. Here it looks like you are adding a new entry to the dict (with the key being `False` or `True`) and setting it's value to None. I.e. you are not replacing the `"-"` as you intend.